lib/lighthouse.js

```diff
@@ -81,28 +110,42 @@ exports.fetchAndSave = function(pwaId) {
  */
 exports.processLighthouseJson = function(lighthouseJson) {
```

Comment:
This method has many instances of the usage of 'var'. We should always use 'const' / 'let'